docs/ref/settings.txt

```diff
@@ -48,13 +48,13 @@ ADMINS
 Default: ``[]`` (Empty list)
 
 A list of all the people who get code error notifications. When
-``DEBUG=False`` and a view raises an exception, Django will email these people
+``DEBUG=False`` and a view raises an exception, Django will send an email to these people
```

Comment:
I think it's fine to use email as a verb? Note that in documentation lines should be wrapped at 79 characters.